services/Cdiscount.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Cdiscount",
+    "documents": {
+        "Commercial Terms": {
+            "fetch": "https://clients.cdiscount.com/simple/cgv.aspx",
+            "select": "div.acDes"
+        },
+        "Terms of Service": {
+            "fetch": "https://i2.cdscdn.com/resources/imagesok/cgu/mkp/cgu_acheteurs_mp_cdiscount.pdf"
+        },
+        "Trackers Policy": {
+            "fetch": "https://www.cdiscount.com/vie-privee-et-cookies.html",
+            "executeClientScripts": true,
+            "select": "div#scContent"
```

Increase robustness.

```suggestion
            "select": "#scContent"
```